<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QWidget" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <x>0</x>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CosyVoice</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SplashScreen {
    background: #1a1412;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="coverWidget">
     <property name="styleSheet">
      <string notr="true">QWidget#coverWidget {
    background: #1a1412;
    border: none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="coverLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="coverLabel">
        <property name="text">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#coverLabel {
    background: #1a1412;
    border: none;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="infoBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#infoBar {
    background: #2a1d19;
    border-top: 1px solid #3d2b25;
}</string>
     </property>
     <layout class="QHBoxLayout" name="infoBarLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>Version 1.0.0</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#versionLabel {
    color: #c4a77d;
    font-size: 10px;
    font-family: 'Courier New', monospace;
    background: transparent;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="separatorLabel">
        <property name="text">
         <string>|</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#separatorLabel {
    color: #6b5c52;
    font-size: 10px;
    background: transparent;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>@3uyuan1ee</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#authorLabel {
    color: #c4a77d;
    font-size: 10px;
    font-family: 'Courier New', monospace;
    background: transparent;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="infoSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
